Functions/QualityControl/templateConfigReportPDFAlternative.json

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
Same for all files, if they are now dynamically created, please add the suffix "example" or something so this is clear.